src/test/playwright/support/pageobjects/exercises/quiz/QuizExerciseCreationPage.ts

```diff
@@ -27,9 +28,34 @@ export class QuizExerciseCreationPage {
         const fileContent = await Fixtures.get('exercise/quiz/multiple_choice/question.txt');
         const textInputField = this.page.locator('.monaco-editor');
         await textInputField.click();
+        await clearTextField(textInputField);
```

Comment:
Interestingly, this line missing was a mistake in the test case. The `fileContent` was getting appended to the default Artemis quiz template, resulting in a malformed quiz.